.github/CODEOWNERS

```diff
@@ -53,40 +53,40 @@ go.sum @harshit-gangal @mattlord @rohit-nayak-ps @systay @frouioui
 /go/vt/vtctl/vtctl.go @notfelineit @rohit-nayak-ps
 /go/vt/vtctl/grpcvtctldclient @notfelineit @mattlord
 /go/vt/vtctl/grpcvtctldserver @notfelineit @mattlord
-/go/vt/vtctl/reparentutil @GuptaManan100
```

Comment:
Should directories with no owner be removed, or another owner assigned?

Reply:
I think no owner is fine for now - we can update this later.